spec/integration/lib/auth0/api/v2/api_stats_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+describe Auth0::Api::V2::Stats do
+  let(:client) { Auth0Client.new(v2_creds) }
+
+  describe '.active_users' do
+    let!(:active_users) { client.active_users }
+    it { expect(Integer(active_users)).to be >= 0 }
+  end
+
+  describe '.daily_stats' do
+    let(:from) { Time.zone.today.prev_day.strftime('%Y%m%d') }
+    let(:to) { Time.zone.today.strftime('%Y%m%d') }
+    let(:daily_stats) { client.daily_stats(from, to) }
+    it { expect(daily_stats.size).to be > 0 }
```

Comment:
This greater than 0 ensures that you get at least one entry in the JSON answered by the API. It doens't matter the amont of users logged in. The response of this endpoint is:
```
[
  {
    "date": "2014-12-14T00:00:00.000Z",
    "logins": 0,
    "signups": 0
  },
  {
    "date": "2014-12-15T00:00:00.000Z",
    "logins": 0,
    "signups": 0
  },
  {
    "date": "2014-12-16T00:00:00.000Z",
    "logins": 0,
    "signups": 0
  },
...
```